### PR TITLE
Connect to API and refactor the code.

### PR DESCRIPTION
**Pull request by franclobo**
### Connect to API.

- Add redux-thunk to the project.
- Use [axios](https://axios-http.com/docs/intro) for making API calls.
- Fetch book list from Bookstore API on Books page load.
- Add an Action Creator, that returns a function.
- Fetch list of books.
- Dispatch another action (add an action type) that will pass API response to reducer and updates the state.
- Refactor the add book and remove book features to persist the changes in the server.
- Change the existing Action Creators, so they return functions.
- Make changes by using API.
- Dispatch an action (action type that existed before refactor) that will updates the state accordingly.

### CODE REVIEW

File: src/redux/books/Books.js [hunk 1]
```diff
@@ -1,61 +1,73 @@
...
-  let newBook = {};
-  if (title.value && author.value) {
-    newBook = {
-      id: uuidv4(),
-      title: title.value,
-      author: author.value,
-    };
+export const loadBooks = () => async (dispatch) => {
+  try {
+    const res = await axios.get('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/lbT5j970vaLoTzIT3rPx/books');
```
Comment (mateo951) [inline src/redux/books/Books.js:16]: All Bookstore API calls are done with axios 🤓

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @franclobo, 

#### Highlights 😎
- Great PR description 🙌
- `redux-thunk` is added to the project as a package and used as a middleware
- API implementation 😁
 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Comment by mateo951** [comment]
Please be sure to add the appropiate [template](https://github.com/microverseinc/readme-template/blob/master/README.md) to your readme file as its a key part of the general requirements 🤓

**Review by danielufeli — APPROVED** [review]
Hi @franclobo ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**